serveur_calcul_python/tests/unit/test_reg_error_cases.py

```
@@ -0,0 +1,107 @@
+import classes.parking_inventory as PI
+import tests.data_gen.erroneous_reg_data_store as ERDS
+
+'''Cases where the slope and intercept are set to all None which is invalid '''
+def test_all_none_values_simple_rule():
+    '''
+    # test_all_none_values
+    Tests whether an error is thrown when all the values are none in the regulation definition
+    '''
+    with pytest.raises(ValueError) as excinfo:
+        park_reg = ERDS.generate_none_simple_regulation()
+        park_reg.validate()
```

Ruff (0.15.20)

[warning] `pytest.raises()` block should contain a single simple statement

(PT012)